content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+---
+reviewers:
+- phanama
+- wahyuoi
+title: Horizontal Pod Autoscaler
+feature:
+  title: Horizontal scaling
+  description: >
+    Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
+
+content_template: templates/concept
+weight: 90
+---
+
+{{% capture overview %}}
+
+Horizontal Pod Autoscaler secara otomatis akan men-*scale* jumlah *pod* didalam kontroler replikasi, *deployment*, 
+*replica set* ataupun *stateful* berdasarkan hasil observasi penggunaan CPU(atau, dengan 
+[*costum* metrik](https://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md), pada beberapa aplikasi yang menyediakan metrik). 
+Perlu dicatat bahwa Horizontal Pod Autoscaling tidak dapat diterapkan pada objek yang tidak dapat di-*scale*, seperti DeamonSets. 
+
+Horizontal Pod Autoscaler diimplementasikan sebagai Kubernetes API *resource* dan sebuah kontroller.
```

Comment:
resource is "sumber daya" as per RSNI glosarium
kontroler is not Indonesian word, should use _controller_
```suggestion
HorizontalPodAutoscaler diimplementasikan sebagai sumber daya Kubernetes API dan sebuah _controller_.
```